service_info_cms/migrations/0008_auto_20160113_1510.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('service_info_cms', '0007_auto_20160113_0017'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
I'm not so religious about this stuff, but I've seen somebody else who works on this codebase spend oodles of time collapsing migrations that have actually been used on production; we're adding 6-7 migrations for unmerged code :)

At this point I think it would be good to delete the added migrations then regenerate minimal ones for your branch.